.cirrus.yml

```diff
@@ -9,3 +11,37 @@ task:
         - node_version: 11.14.0
   install_script: npm ci
   test_script: npm test
+
+lint_task:
+  install_script: npm ci
+  test_script: npm run lint
+
+test_task:
+  container:
+    matrix:
+      image: node:12
+      image: node:11
+      image: node:10
+      image: node:8
+  install_script: npm ci
+  test_script: npm test
+
+coverage_task:
+  script:
+    - npm ci
+    - npm test
+    - npm run codecov
+
+publish_task:
+  only_if: $CIRRUS_TAG =~ '^v.*'
+  depends_on:
+    - test_task
+    - lint_task
+    - windows_task
+  environment:
+    NPM_TOKEN: ENCRYPTED[188593d2219e49e28e05c983a768a7e89ef89441f07f450a58744161674b2e7f127caff0a8021bb5bb75a09d4e18750f]
```

Comment:
Does this have a race where both Circle and Cirrus would race to do a publish?

Reply:
This PR deletes the CircleCI config... so I don't think so